src/main/controllers/models/country.js

```diff
@@ -1,4 +1,4 @@
-const countriesList = require('./../../../assets/data/country-list.json');
+const countriesList = require('../../assets/data/country-list.json');
 
 const Promise = require('bluebird');
 
```

Comment:
We can delete all the .init functions in here as they are replaced by knex migration file

Reply:
@altninja for all models?

Reply:
should be - I'll double check it though